SUPPORT_POLICY.rst

```diff
@@ -22,16 +22,13 @@ This table describes the current support status of each major version of the AWS
       - Next status
       - Next status date
     * - 1.x
-      - Maintenance
       - End of Support
-      - 2022-07-08
     * - 2.x
-      - Generally Available
-      - Maintenance
-      - 2021-07-13
+      - End of Support
+      -
+      -
```

Comment:
We should have updated this on `2021-07-13`
and set the Next Status to End of Support.
But I think the dates are ok for now.